nautobot_chatops/workers/nautobot.py

```diff
@@ -422,16 +377,24 @@ def get_interface_connections(dispatcher, filter_type, filter_value_1, filter_va
                 f'Unknown filter type "{filter_type}"',
             )  # command did not run to completion and therefore should not be logged
 
-        if filter_type != "device":
+        if filter_type != "device" or menu_item_check(filter_type):
             dispatcher.prompt_from_menu(
-                f"nautobot get-interface-connections {filter_type}", f"Select a {filter_type}", choices
+                f"nautobot get-interface-connections {filter_type}",
+                f"Select a {filter_type}",
+                choices,
+                offset=menu_offset_value(filter_type),
             )
         else:
-            dispatcher.prompt_from_menu(f"nautobot get-interface-connections {filter_type}", "Select a site", choices)
+            dispatcher.prompt_from_menu(
+                f"nautobot get-interface-connections {filter_type}",
+                "Select a site",
+                choices,
+                offset=menu_offset_value(filter_type),
```

Comment:
This seems wrong to me - if you get to the `else` case, here, you've already established that `filter_type == "device"`. Should any of the above block be using `filter_value_1` instead of `filter_type`?

Reply:
Great catch, thank you.  I fixed this in the latest commit.
Also while troubleshooting this, I came across 2 more bugs.  I'll put them in a separate issue/PR.